ui/src/app/machines/views/MachineList/MachineListTable/MachineListTable.test.tsx

```diff
@@ -480,13 +482,11 @@ describe("MachineListTable", () => {
         wrapper
           .find("[data-test='group-cell'] input")
           .at(0)
-          .props()
-          .className.includes("p-checkbox--mixed")
+          .hasClass("p-checkbox--mixed")
```

Comment:
How have I not heard of `hasClass` before!?

Reply:
Yeah I keep forgetting about it too.